src/core.js

```diff
@@ -23,15 +23,15 @@ LICENSE: GPL-2.0
   - Listen to changes in sync storage
 **/
 
-const MAX_BUTTON_ITEMS = 999;
+const MAX_BUTTON_ITEMS = 9;
```

Comment:
It should be 999

Reply:
Yes, sorry, just forgot to press Ctrl+S in VS Code :)

Reply:
Please take a look again, I changed the last commit.